packages/cron/src/jobs/names.js

```diff
@@ -1,5 +1,5 @@
 import debug from 'debug'
-import retry from 'p-retry'
+import { pRetry, retry } from 'p-retry'
```

Comment:
As far as I can see, `retry` isn't even exported from the 'p-retry' module? 🤔 But it seems that `pRetry` is for promise/async functions, and `retry` is for sync functions. So I guess it makes things a bit clearer to only use `pRetry` when we're actually dealing with async stuff..?

Reply:
Yes, let's only use pRetry here 👍

Reply:
I've now added another commit that switches to `pRetry` for retrying the `db.listNames` method. And I've switched to using `async` and `await` in that whole call chain, because I think it makes it clearer what's going on.

Quick re-review?